src/test/kotlin/com/zepben/evolve/streaming/data/SetCurrentStatesStatusTest.kt

```diff
@@ -82,7 +83,7 @@ class SetCurrentStatesStatusTest {
     }
 
     @Test
-    fun `StateEventFailure from protobuf and then back to protobuf`() {
+    internal fun `StateEventFailure from protobuf and then back to protobuf`() {
```

Comment:
todo: add `testStateEventFailure` call near the bottom of this test to cover the case where `pb.reasonCase == PBStateEventFailure.ReasonCase.UNSUPPORTEDMRID`